VS.Common/Settings/ChutzpahUTESettings.cs

```diff
@@ -52,10 +53,25 @@ public bool EnabledTracing
             }
         }
 
+        [Browsable(true)]
+        [Category("UTE")]
+        [DisplayName("Browser Arguments")]
+        [Description("The arguments used to pass to the browser being used for the execution of the test.")]
+        public string BrowserArguments

```

Comment:
I am moving away from adding settings in the VS dialogs. Please add to the chutzpah.json file instead.